zkchannels-crypto/Cargo.toml

```diff
@@ -23,3 +23,4 @@ serde-big-array = { version = "0.3.2", features = ["const-generics"] }
 sha3 = "0.9.1"
 sqlx = { version = "0.5.2", features = ["runtime-tokio-rustls"], optional = true }
 thiserror = "1"
+chrono = "0.4"
```

Comment:
Per discussion with @yaymukund: there is not a separate compile tag for example-only dependencies. Chrono has a lot of features so I can make sure I'm only using the ones I want. There are some ways to only compile it in certain cases, like workspaces (?) or feature flags, but that might be overkill for this situation.

Recommendation is check how long it takes to compile. If it's a long time, consider limiting the features used or rewriting the example to use our own fake time API.

Reply:
Can you put this in `[dev-dependencies]`? 

> Dev-dependencies are not used when compiling a package for building, but are used for compiling tests, examples, and benchmarks.

from https://doc.rust-lang.org/cargo/reference/specifying-dependencies.html#development-dependencies

Reply:
Oh, thank you. For some reason I thought this was only for test dependencies, but I will move it here.